Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM php:alpine3.10
+
+WORKDIR /workdir
+
+RUN apk add composer
+
+COPY composer.json composer.json
+RUN composer install
+
+RUN adduser -u 2004 -D docker
+COPY docs /docs
+RUN chown -R docker:docker . /docs
+
+USER docker
+
+COPY src src
+
+ENTRYPOINT [ "php", "-d", "memory_limit=-1" ]
```

Comment:
are we sure of:
`memory_limit=-1` ?

Reply:
This should be the way you tell php to use all the memory of the container.

Reply:
> Note that to have no memory limit, set this directive to -1

https://www.php.net/manual/en/ini.core.php

it's not clear to me if this is bounded to the machine memory or totally unbounded.